src/build/browser/browser-static/bootstrap.js

```diff
@@ -134,11 +92,39 @@ function bootstrap(initialImport, target) {
 
 	var loaded = false;
 	w._continueLoad = function() {
-		if (!loaded) {
-			loaded = true;
-			var el = d.createElement('script');
-			el.src = target + '.js';
-			d.getElementsByTagName('head')[0].appendChild(el);
+		var doThings = function() {
+			if (!loaded) {
+				loaded = true;
+				// Include the game code
+				var el = d.createElement('script');
+				el.src = target + '.js';
+				d.getElementsByTagName('head')[0].appendChild(el);
+			}
+		};
+
+		var _continueLoadDefer;
+
+		w.addEventListener('message', function(event) {
+			if (event.data === 'partialLoadContinue') {
+				if (_continueLoadDefer) {
+					_continueLoadDefer.resolve();
+					_continueLoadDefer = undefined;
+				}
+			}
+		});
+		// Preload suggestions and then tell the parent bootstrapping is complete
+		jsio.__env.preloadModules(function() {
+			w.parent.postMessage('bootstrapping', '*');
+		});
+
+		var partialLoadKey = jsio.__env.getNamespace('partialLoad');
+		if (localStorage && localStorage.getItem(partialLoadKey)) {
+			localStorage.removeItem(partialLoadKey);
```

Comment:
As far as I can tell, your proposed changes *make* jsio reliant on devkit functionality since it makes `jsio.js` getNamespace look for the global `CONFIG` and/or look for `apps/` in the URL, both of which are state provided by devkit only.  I'm proposing the alternative -- js.io *doesn't* depend on devkit, and any local storage namespacing or caching lives *entirely* in devkit -- no `getNamespace` calls from devkit to js.io that depend on devkit variables/state.